plugins/private/qVoxFall/include/qVoxFall.h

```diff
@@ -0,0 +1,55 @@
+//##########################################################################
+//#                                                                        #
+//#                     CLOUDCOMPARE PLUGIN: qVoxFall                      #
+//#                                                                        #
+//#  This program is free software; you can redistribute it and/or modify  #
+//#  it under the terms of the GNU General Public License as published by  #
+//#  the Free Software Foundation; version 2 of the License.               #
+//#                                                                        #
+//#  This program is distributed in the hope that it will be useful,       #
+//#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+//#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         #
+//#  GNU General Public License for more details.                          #
+//#                                                                        #
+//#                 COPYRIGHT: THE UNIVERSITY OF NEWCASTLE                 #
+//#                                                                        #
+//##########################################################################
+
+#pragma once
+
+#ifndef Q_VOXFALL_PLUGIN_HEADER
```

Comment:
Wherever there's a _#pragma once_ statement, you don't need the _#ifnded Q_XXX + #define Q_XXX + #endif_ statements (I think it applies to pretty much all header files?)